docs/dls/component-design-guidelines.md

```diff
@@ -0,0 +1,100 @@
+---
+id: component-design-guidelines
+title: Component Design Guidelines
+description: Documentation on Design
+---
+
+Be it a new component contribution, or plugin specific components, you'll want
+to follow these guidelines, we'll cover the 3 main subjects that define the
+general look and feel of your components, all of which build on top of the MUI
+theme features:
+
+- Layout
+- Color palette
+- Typography
+
+## 🏗️ Layout
+
+Layout refers to how you organize or stack content, whenever possible we want
+to use Backstage's components (check the [Storybook][1] for a list and demo)
+or MUI components (check the [MUI docs][2]).
+
+If none of these fit your layout needs, then you can build one, however using
+HTML+CSS directly is not recommended, it's better to use MUI layout components
```

Comment:
```suggestion
HTML+CSS directly is not recommended: it's better to use MUI layout components
```

Reply:
Not sure if this one makes sense, I'm not a native english speaker but is putting a colon `:` in there syntactically correct?

Reply:
Changed, let me know if `:` is better than `;`